packages/calcite-components/src/components/tab/tab.tsx

```diff
@@ -69,7 +69,7 @@ export class Tab {
           role="tabpanel"
           tabIndex={this.selected ? 0 : -1}
         >
-          <section>
+          <section class={CSS.content}>
```

Comment:
Not for this PR, but it'd be good to replace all selectors using `section` with the `content` class.